usersimcrs/run_simulation.py

```diff
@@ -70,13 +72,17 @@ def main(config: confuse.Configuration, agent: Agent) -> None:
     # Loads domain, item collection, and preference data
     domain = Domain(config["domain"].get())
 
-    item_collection = ItemCollection()
-    item_collection.load_items_csv(
-        config["items"].get(),
-        domain=domain,
-        domain_mapping=config["domain_mapping"].get(),
-        id_col=config["id_col"].get(),
+    item_collection = ItemCollection(
+        config["collection_db_path"].get(), config["collection_name"].get()
     )
+    if config["items"].get():
```

Comment:
Nit: should we check if all the necessary fields exist in the config file before running `load_items_csv`?

Reply:
All the necessary fields are defined in the default configuration that is automatically loaded at the beginning of the run.